docs/api/helpers.md

```
@@ -0,0 +1,452 @@
+        print(f"Operation '{operation_name}' took {duration:.3f} seconds")
+
+# Usage
+async def monitored_operation():
+    async with monitor_performance("database_query"):
+        await asyncio.sleep(0.5)  # Simulate work
+        return "Query result"
+```
+
+## Best Practices
+
+### 1. Error Handling
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)